backend/src/data/models/podcast/podcast.model.ts

```diff
@@ -23,7 +23,7 @@ class Podcast extends Abstract {
 
   [PodcastDTOKey.TYPE]: PodcastType;
 
-  [PodcastDTOKey.GENRE_ID]: number | null;
+  [PodcastDTOKey.GENRE_ID]: number;
```

Comment:
```suggestion
  [PodcastDTOKey.GENRE_ID]: number;
```
shouldn't be changed